CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 * [#1227](https://github.com/ruby-grape/grape/pull/1227): Store `message_key` on Grape::Exceptions::Validation - [@stjhimy](https://github.com/sthimy).
 * [#1232](https://github.com/ruby-grape/grape/pull/1232): Helpers are now available inside `rescue_from` - [@namusyaka](https://github.com/namusyaka).
+* [#1237](https://github.com/ruby-grape/grape/pull/1237): Given now takes multiple inputs and behaves as though the scopes were nested in the inputted order - [@ochagata](https://github.com/ochagata)
```

Comment:
Given is a keyword, so maybe "Allow multiple parameters in `given`, which behaves as if the scopes were nested in the inputted order".
 
Missing a trailing period, please.